make/targets/openshift/golangci-lint.mk

```diff
@@ -0,0 +1,52 @@
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+)
+
+GOLANGCI_LINT_VERSION ?=1.42.1
+GOLANGCI_LINT ?=$(PERMANENT_TMP_GOPATH)/bin/golangci-lint-$(GOLANGCI_LINT_VERSION)
+
+golangci_lint_downloaded_filename := golangci-lint-$(GOLANGCI_LINT_VERSION)-$(GOHOSTOS)-$(GOHOSTARCH)
```

Comment:
GOHOSTOS and GOHOSTARCH are not set on my system, will this work?

Reply:
aaaah... good point.. I have those in my local box. Will correct this.